src/SolverInterface/lpqp_to_conic.jl

```diff
@@ -239,6 +248,27 @@ for f in [:optimize!, :status, :getsolution, :getobjval, :getvartype]
     @eval $f(model::LPQPtoConicBridge) = $f(model.lpqpmodel)
 end
 
+function getdual(model::LPQPtoConicBridge)
+    if model.qcp
+        error("getdual not supported for QCP")
```

Comment:
Just a nitpick, but people using this interface are thinking about conic problems. We should refer to ``SOC`` and ``SOCRotated`` instead of "QCP" in the error messages.